cli/cliutils/cliutils.go

```diff
@@ -1164,7 +1164,27 @@ func ExchangeGet(service string, urlBase string, urlSuffix string, credentials s
 
 	resp := InvokeRestApi(httpClient, http.MethodGet, url, credentials, nil, service, apiMsg)
 	defer resp.Body.Close()
-	bodyBytes, err := ioutil.ReadAll(resp.Body)
+
+	respBody := io.Reader(resp.Body)
+	if resp.Header.Get("Content-type") == "application/octet-stream" {
+		// Show progress of binary files downloading
+		msgPrinter.Print("Downloading object")
+		chunkNumber := 0
+		respBody = &progressReader{resp.Body, func(r int) {
+			chunkNumber++
+			if r > 0 {
+				// Show progress every 5 chunks to make the loading line shorter
+				if chunkNumber%5 == 0 {
+					msgPrinter.Print(".")
```

Comment:
This string does not need to be translated, so please just use fmt.Print.

Reply:
Updated with using of fmt.Print